olp-cpp-sdk-authentication/src/SignInResultImpl.cpp

```diff
@@ -91,5 +111,70 @@ const std::string& SignInResultImpl::GetScope() const { return scope_; }
 
 bool SignInResultImpl::IsValid() const { return is_valid_; }
 
+std::string SignInResultImpl::ParseJwtToken(const std::string& token) {
+  const std::string::size_type first_dot = token.find_first_of('.');
+  if (first_dot == std::string::npos) {
+    OLP_SDK_LOG_ERROR_F(kLogTag, "Cannot parse ClientId. Wrong token format.");
```

Comment:
```suggestion
    OLP_SDK_LOG_ERROR(kLogTag, "Cannot parse ClientId. Wrong token format.");
```
Not critical, but no need to add formatted logging. Same with other OLP_SDK_LOG... in this file